transform/app/tasks/transform/dump/transform.py

```diff
@@ -42,28 +53,54 @@ def transform_data(prev_task_status: Optional[CeleryTaskStatus]) -> CeleryTaskSt
 
                     for file_num, file_path in enumerate(tqdm(paths)):
                         try:
-                            data = load_file_from_s3(file_path, s3_client)
-                            logger.info("loaded: %s", file_path)
-                            df = load_request_data(
-                                spark, data, input_schema, collection_name
+                            df_transformed = transform_file(
+                                file_path,
+                                transformer,
+                                spark,
+                                s3_client,
+                                collection_name,
+                                input_schema,
+                                error_log,
+                            )
+
+                            if df_transformed is None:
+                                logger.warning(
```

Comment:
its rather an error.
I would rather write: 
`"%s file in %s collection failed to be transformed`

Reply:
You are right, that's shorter and more intuitive